packages/taquito-contracts-library/src/rpc-wrapper.ts

```diff
@@ -225,6 +226,12 @@ export class RpcWrapperContractsLibrary implements RpcClientInterface {
   ): Promise<PreapplyResponse> {
     return this.rpc.runOperation(op, { block });
   }
+  async simulateOperation(
+    op: RPCSimulateOperationParam,
+    { block }: RPCOptions = defaultRPCOptions
+  ): Promise<PreapplyResponse> {
```

Comment:
surprised to see PreapplyReponse here, but understand if it's returning the same object

Reply:
PreapplyResponse is basically just `content` so I guess it was just reusing the type. It was the same for run_operation so I just followed suit.